docs/resources/ocean_aks_np.md

```diff
@@ -276,3 +276,35 @@ The following arguments are supported:
     * `min_nics` - (Optional) Minimum number of network interfaces.
     * `min_disk` - (Optional) Minimum number of data disks available.
     * `vm_types` - (Optional, Enum `"generalPurpose", "memoryOptimized", "computeOptimized", "highPerformanceCompute", "storageOptimized", "GPU"`) The filtered vm types will belong to one of the vm types from this list.
+
+<a id="update-policy"></a>
+## Update Policy
+
+* `update_policy` - (Optional)
+  * `should_roll` - (Required) Enables the roll.
+  * `conditioned_roll` - (Optional, Default: false) Spot will perform a cluster Roll in accordance with a relevant modification of the cluster’s settings. When set to true , only specific changes in the cluster’s configuration will trigger a cluster roll (such as AMI, Key Pair, user data, instance types, load balancers, etc).
```

Comment:
such as examples are same as ocean aws, please modify according to ocean aks

Reply:
Updated as per AKS